### PR TITLE
New PR - Audit & fix Information.md

### PR DESCRIPTION
**Pull request by Yoda-Canada**
I will closed the PR #101 and open the new PR for Audit & fix Information.md page #44

Steps done to modify and check Information.md:

Fixed images work in dark modes. The memory model image's downward arrows and the letter in the bottom right corner are not clear in dark modes. I wrapping them in a tag and adding class "mdImg" to it.
The images can't display in all models. Change all the image path from "/img/.jpg" to "../../static/img/.jpg". The issue is fixed.
Fix Frontmatter for the page to include id, title, slug, and description.
Replace blockquote with Admonitions for clarity.
Added alt text for diagram images.
Some of " * "will get converted into _ after being formatted with Prettier. I use * replace them.

### CODE REVIEW

**Comment by Yoda-Canada** [comment]
@humphdopen I open a new PR for Audit & fix Information.md page #44  and I will closed the PR #101. I'm sorry it's inconvenient for you

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@
 # Generated files
 .docusaurus
 .cache-loader
+yarn.lock
```
Comment (humphd) [inline .gitignore:10]: Revert this change please.

**Review by humphd — CHANGES_REQUESTED** [review]
Please undo the change to yarn.lock in this PR, it's not needed for these changes.

Only `git add` the files you are modifying, don't use `git add .`

File: .gitignore (hunk 1 above)
Comment (Yoda-Canada) [reply .gitignore:10]: How to undo the change? I use ctrl+z, but can't

Comment (humphd) [reply .gitignore:10]: Checkout the file you want from the commit on the `main` branch:

`git checkout main .gitignore`
`git checkout main yarn.lock`

Comment (Yoda-Canada) [reply .gitignore:10]: I also need to ` git checkout main docs/A-Introduction/information.md `
       and  ` git checkout main static/img/ram.pgn ` , right?

Comment (Yoda-Canada) [reply .gitignore:10]: then only add information.md and ram.pgn, right?

Comment (Yoda-Canada) [reply .gitignore:10]: From the beginning of the project. I create a new branch isssue-44, then changed information. md,  ram.pgn, .gitignore and yarn.block. then add commit all the four files. The last step is ` git push origin issue-44 `.

Comment (Yoda-Canada) [reply .gitignore:10]: I didn't merge the issue-44 to the main. Should I use this command ` git checkout issue-44 yarn.lock `?

Comment (TueNguyen2911) [reply .gitignore:10]: Seems like you have `.gitignore` and `yarn.lock` staged in git. Use `restore --staged <file>` to unstage them, use `git checkout issue044 <file>` to rollback their content. Then use `git add <file>` for files that you are meant to change, in this case only `docs/A-Introduction/information.md` and maybe files in `static/img`. Rebase it and force push.
